test/functionalTest/cases/2769_invalid_regular_expression_for_idPattern/invalid_regular_expression_for_idPattern.test

```diff
@@ -0,0 +1,93 @@
+# Copyright 2017 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Two alternatives:

* Throw the PR away
* Throw the invalid_regular_expression_for_idPattern.test away, after moving its step 2 to crash_with_invalid_subscription_when_matching.test